config/webpack.config.karma.js

```diff
@@ -3,21 +3,42 @@ const path = require('path');
 
 // These modifications are required to have proper coverage with karma-coverage-istanbul-reporter.
 webpackConfig.devtool = 'inline-source-map';
-webpackConfig.module.rules.find(rule => rule.loader === 'ts-loader').options.compilerOptions = {
-    module: 'commonjs',
-    inlineSourceMap: true,
-    sourceMap: undefined,
-    outDir: undefined
-};
-webpackConfig.module.rules.push({
-    enforce: 'post',
-    test: /\.ts$/,
-    loader: 'istanbul-instrumenter-loader',
-    exclude: path.resolve('tests/'),
-    query: {
-        esModules: true
+webpackConfig.module.rules = [
+    {
+        test: /\.ts$/,
+        use: [
+            {
+                loader: 'ts-es5-istanbul-coverage'
```

Comment:
Le premier loader deviens l'instrumenteur ?

Reply:
Check: https://www.npmjs.com/package/ts-es5-istanbul-coverage
tl;dr: it runs second. so it instrument the instrumented